[WIP] headerをsassる #24

mamiline6 opened the pull request.

mamiline6 committed:
html作成

geckotang commented on ablog_sass/app/index.html:
```diff
@@ -0,0 +1,32 @@
+	<ul class="menu">
+		<li class="menu_item"><a href="#" class="menu_link">ページ1</a></li>
+		<li class="menu_item"><a href="#" class="menu_link">ページ2</a>
+			<ul class="menu_innermenu">
```
`menu_innermenu` を `innermenu` にするかな〜僕だったらね。

geckotang commented on ablog_sass/src/scss/style.scss:
```diff
@@ -0,0 +1,12 @@
+	#header
+--------------- */
+#header {}
+.meue.trg {
```
めうえ

mamiline6 committed:
タスク追加（ブラウザーシンク）

mamiline6 committed:
browserSync タスク見直し

パス変更してないのに Cannot GET /  で動かなくなったため
- [] html,scss保存で同期するけど、ターミナル側で[error] You tried to start Browsersync twice! To create multiple instances, use browserSync.create().init()

mamiline6 committed:
.innermenu にクラス名変更

mamiline6 committed:
不要コード削除

mamiline6 committed:
scssで書く

- クラス名を関連するグループにしてなるべくまとめる
- [ ] サブページのulの表示されない問題

mamiline6 committed:
cleafix 追加

mamiline6 committed:
overflow: hidden; 削除

mamiline6 committed:
clearfix 追加

mamiline6 commented:
@geckotang 

コードみてもらって、ありがとうございます！

mamiline6 committed:
gulp タスク追加（gulp-uglify）

mamiline6 committed:
タスクの追加（jsの圧縮）
